spring-xd-dirt/src/main/java/org/springframework/xd/dirt/integration/bus/converter/ByteArrayToStringMessageConverter.java

```diff
@@ -65,7 +66,7 @@ public ByteArrayToStringMessageConverter() {
 	 * Don't need to manipulate message headers. Just return payload
 	 */
 	@Override
-	public Object convertFromInternal(Message<?> message, Class<?> targetClass) {
+	protected Object convertFromInternal(Message<?> message, Class<?> targetClass, Object conversionHint) {
```

Comment:
I followed the same pattern used for core spring subclasses of AMC. The third argument is never used in these.